dotcom-rendering/src/components/ArticleMeta.stories.tsx

```diff
@@ -448,15 +356,16 @@ export const ImmersiveStory: StoryObj = ({ format }: StoryArgs) => {
 	);
 };
 ImmersiveStory.storyName = 'Immersive';
-ImmersiveStory.decorators = [lightDecorator([immersiveFormat])];
-ImmersiveStory.args = { format: immersiveFormat };
+ImmersiveStory.decorators = [
```

Comment:
```suggestion
Immersive.decorators = [
```

Reply:
The story name can be inferred automatically from the component name, so where redundant I removed it. Happy to keep it if people are keen!